frontend/src/helpers/regexValidator.js

```diff
@@ -0,0 +1,39 @@
+function validateName(string) {
```

Comment:
All of these functions are perfect candidates for unit tests - easy to check for expected input / output of expected cases.

Reply:
I'll add tests for these new components in another PR, because this PR is already big enough.